### DISCUSSION TITLE
Translucent/transparent card themes

### DISCUSSION THREAD

**Discussion by GermanBartoli** [anuraghazra/github-readme-stats#1920] [Q&A] [answered]
This is the most similar but it would be nice to have a translucent card with no background

![image](https://user-images.githubusercontent.com/53313625/182506304-8be51b32-1ad1-4e68-9537-dc9ff1fe6f33.png)

**Comment by rickstaa** [comment, marked as answer]
Hey, @bartoligerman497, thanks for your feature request. We already support 8 digit hex codes, so you can already use a transparent background:

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&bg_color=ffffff00&text_color=0055ff)](https://github.com/anuraghazra/github-readme-stats)
```

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&bg_color=ffffff00&text_color=0055ff)](https://github.com/anuraghazra/github-readme-stats)

You, however, then have to make sure that the text colour is readable. A better way is to use a dynamic theme (see https://github.com/anuraghazra/github-readme-stats/discussions/1415#discussioncomment-2152317 or my [README ](https://github.com/rickstaa/rickstaa) for examples).

**Comment by rickstaa** [comment]
All the currently available ways for creating a dynamic theme are now documented in the [README.md](https://github.com/anuraghazra/github-readme-stats#responsive-card-theme) (See https://github.com/anuraghazra/github-readme-stats#responsive-card-theme). GRS now also includes a [transparent theme](https://github.com/anuraghazra/github-readme-stats#use-the-transparent-theme).
